edgelet/iotedge-proxy/docker/linux/amd64/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine:3.9
```

Comment:
Should we run this proxy as a less privileged user?

Reply:
Good idea. Make the `proxy` user instead root